.github/CONTRIBUTING.md

```diff
@@ -29,32 +29,32 @@ Our central development branch is master. Coding is done on feature branches bas
 	```
 2. Commit and push code to your branch:
 
-    - Commits should be self-contained and contain a descriptive commit message.
-    - Please make sure your code is well-formatted and adheres to PEP8 conventions (for Python) and the `.eslint` located in the repository (for JavaScript).
+    - Commits should be self-contained and they must contain a descriptive commit message.
+    - Please make sure your code is well-formatted and adheres to 'PEP8' conventions (for Python) and the `.eslint` located in the repository (for JavaScript).
     - Please ensure that your code is well tested.
-    - We highly encourage to use `flake8` to follow the PEP8 styling. Run the following command before creating the pull request:
+    - We highly encourage to use `flake8` to follow the 'PEP8' styling. Run the following command before creating the pull request:
 	```
 	flake8 ./
 	```
-    - Also, for build errors and checking adherence to `.eslint`, run the following command during development:
+    - Also, for the build errors and to check adherence to `.eslint`, run the following command during development:
 	```
 	webpack --progress --colors --watch
 	```            
-    - If you are using Sublime Text, you can lint code while writing it. Install the following plugins for Sublime Text.
+    - If you are using a Sublime Text, you can lint the code while writing it. Install the following plugins for a Sublime Text.
         ```
         SublimeLinter
         SublimeLinter-contrib-eslint
         SublimeLinter-flake8
         ```
-        *For installing the Sublime Package Control Manager in Sublime-Text Editor use [this](https://packagecontrol.io/installation#st2) link.*
+        *For installing the Sublime Package Control Manager in a Sublime-Text Editor use the link: (https://packagecontrol.io/installation#st2) .*
```

Comment:
```suggestion
        *For installing the Sublime Package Control Manager in Sublime-Text Editor, use this [link](https://packagecontrol.io/installation#st2).*
```
Square brackets are used to create hyperlinks in markdown. You can learn more about it [here](https://guides.github.com/features/mastering-markdown/).